server/src/main/java/org/spine3/server/aggregate/EventApplierMethod.java

```diff
@@ -49,12 +52,22 @@
     }
 
     /**
+     * Returns the set of the event classes generated by the passed {@code Aggregate} class.
+     *
+     * @param clazz the class of the aggregate
+     * @return immutable set of event classes
+     */
+    static ImmutableSet<Class<? extends Message>> getEventClasses(Class<? extends Aggregate> clazz) {
```

Comment:
I'm not sure that this is `EventApplierMethod` responsibility. Maybe we should extract this into a utility class or leave in the `Aggregate` class.